contracts/WitnetProxy.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.5;
+
+import "./utils/Upgradable.sol";
+
+contract WitnetProxy {
+  Upgradable public wrb;
+
+  constructor () {}
+
+  receive() external payable {
+    revert("WitnetProxy: no ETH accepted");
+  }
+
+  // solhint-disable-next-line
+  fallback() external payable {
+    address _wrb = address(wrb);
+    assembly {
```

Comment:
:warning: assembly :warning: 

I would appreciate a comment explaining why is this assembly block needed and why it is safe.

Reply:
It will be done once merged all related PRs get merged into a new and single branch (e.g. "v0.3")